starlette/templating.py

```diff
@@ -43,7 +43,7 @@ def __init__(
     async def __call__(self, scope: Scope, receive: Receive, send: Send) -> None:
         request = self.context.get("request", {})
         extensions = request.get("extensions", {})
-        if "http.response.debug" in extensions:
+        if "http.response.debug" in extensions:  # pragma: no branch
```

Comment:
This can be tested without the `TestClient`.

We have some tests that don't use the `TestClient`.

Reply:
That's right, I didn't think about it. 

Added a test to the case based on other similar tests

Reply:
I put this back. I think this is enough for the `"http.response.debug"`.